UmbracoFood.Core/Models/POCO/OrderPoco.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using Umbraco.Core.Persistence;
+using Umbraco.Core.Persistence.DatabaseAnnotations;
+
+namespace UmbracoFood.Core.Models.POCO
```

Comment:
to poco to "plain old com object", a tu chyba bardziej pasuje dto. pozatym moze olac tę końcówkę i zostawić tylko Order i wywalić to do innego namespace'a?